### ISSUE TITLE
🤫 Make OneText untraceable from Shopify

### ISSUE THREAD

**Issue by bluepnume** [one-text/onetext-roadmap#66]
- [ ] Use a different webhook url (admin configurable?)
- [x] Remove onetext specific header

**Comment by jfudem** [comment]
For consideration, we might want to use a variety of different urls to limit contagion.

**Comment by bluepnume** [comment]
True, although we will only ever have a finite number of SLDs -- but we can randomly generate subdomains.
